po/es.po

```diff
@@ -564,49 +578,61 @@ msgstr ""
 "sobre la imagen aumentada en la mesa de luz. el archivo txt tendrá que "
 "existir al momento de importar o el rastreador tendrá que estar activado"
 
-#: ../build/src/preferences_gen.h:2803
+#: ../build/src/preferences_gen.h:2873
 msgid "show scrollbars for center view"
 msgstr "mostrar barras de desplazamiento para la vista central"
 
-#: ../build/src/preferences_gen.h:2814
+#: ../build/src/preferences_gen.h:2884
 msgctxt "preferences"
 msgid "no scrollbars"
 msgstr "sin barras de desplazamiento"
 
-#: ../build/src/preferences_gen.h:2819 ../build/src/preferences_gen.h:2839
+#: ../build/src/preferences_gen.h:2889 ../build/src/preferences_gen.h:2909
 msgctxt "preferences"
 msgid "lighttable"
 msgstr "mesa de luz"
 
-#: ../build/src/preferences_gen.h:2824
+#: ../build/src/preferences_gen.h:2894
 msgctxt "preferences"
 msgid "lighttable + darkroom"
 msgstr "mesa de Luz y Cuarto Oscuro"
 
-#: ../build/src/preferences_gen.h:2842
+#: ../build/src/preferences_gen.h:2912
 msgid "defines where scrollbars should be displayed"
 msgstr "define cuándo mostrar las barras de desplazamiento"
 
-#: ../build/src/preferences_gen.h:2850
+#: ../build/src/preferences_gen.h:2920
+msgid "always show panels' scrollbars"
+msgstr "mostrar siempre los páneles de barras de deplazamiento"
```

Comment:
The correct translation would be:
"mostrar siempre las barras de desplazamiento de los paneles"

Reply:
Thank you! It was corrected.